.gitignore

```diff
@@ -118,8 +118,9 @@ dagit_run_logs
 
 python_modules/libraries/dagster-aws/dagster_aws/ecs/config.yaml
 
-python_modules/dagit/node_modules/
-python_modules/dagit/yarn.lock
```

Comment:
any harm in keeping these for now? I imagine that having these large dirty directories become no longer ignored may result some annoyance / accidents

Reply:
good point